models/comments.models.js

```diff
@@ -42,4 +42,26 @@ exports.insertComment = ({username,body},article_id) => {
         }
         return articleResults.rows[0];
     })
+}
+
+exports.removeComment = (comment_id) => {
+    const queryValues = []
+    const promiseArray = []
+    let sqlString = `DELETE FROM comments`
+    if(comment_id){
+        sqlString += ` WHERE comment_id=$1`
+        queryValues.push(comment_id)
+        promiseArray.push(checkIfCommentExists(comment_id))
```

Comment:
Do you need to check if comment exists? Could you derive this information after the fact?